test/unit/ndr_error/log_test.rb

```diff
@@ -245,18 +235,15 @@ def setup
     end
 
     test 'should auto-calculate clock drift on create when configured' do
-      begin
-        # For DB clock drift checking:
-        NdrError.database_time_checker = lambda do
-          stamp = ActiveRecord::Base.connection.execute('select CURRENT_TIMESTAMP')[0][0]
-          Time.zone.parse(stamp) + Time.zone.utc_offset
-        end
+      test_checker = lambda do
+        stamp = ActiveRecord::Base.connection.execute('select CURRENT_TIMESTAMP')[0][0]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.